### ISSUE TITLE
cargo update breaks versionize_derive compilation.

### ISSUE THREAD

**Issue by kaldonir** [firecracker-microvm/versionize_derive#19]
I have a project which has `versionize_derive` v0.1.4 as a dependency. According to my Cargo.lock, this crate currently depends on `syn` v1.0.109. Since the release of [`syn` v2.0.0](https://github.com/dtolnay/syn/releases) a few days ago, `cargo update` updates `syn` to v2.0.4, which breaks the build of `versionize_derive`. 

In the 0.1.4 release of `versionize_derive`, `Cargo.toml` contains this line:
```toml
syn = { version = ">=1.0.13", features=["full", "extra-traits"]}
```

On `main`, this has been changed to:
```toml
syn = { version = "1.0.13", features=["full", "extra-traits"]}
```
But this change has never been published to crates.io.

In fact, a `versionize_derive` dependency in my `Cargo.toml` pointing to the main branch is correctly handled by `cargo update`
```toml
versionize_derive = { git = "https://github.com/firecracker-microvm/versionize_derive", branch = "main" }
```

Is there anything I can do about this on my side, or is this a bug with `versionize_derive` v0.1.4?

**Comment by blitz** [comment]
@aghecenco @dianpopa Is it possible to release a new version with the fix?

**Comment by kaldonir** [comment]
This has been fixed with v0.1.5 on crates.io, thanks!
